examples/peripherals/uart_async_rxtxtasks/README.md

```diff
@@ -0,0 +1,9 @@
+UART asynchronous example, that uses separate RX and TX tasks
+=============================================================
+
+Starts two FreeRTOS tasks:
+  - One task for transmitting 'Hello world' via the UART.
+  - One task for receiving from the UART.
+
+If you'd like to see your ESP32 receive something, simply short the RX
+and TX pins, defined in ./main/uart_async_rxtxtasks_main.c.
```

Comment:
I suggest using directly the names as in the example code, e.g "...simply short the RX and TX pins, defined in ./main/uart_async_rxtxtasks_main.c. as `TXD_PIN` and `RXD_PIN`".

Reply:
Thanks. I changed it in commit f1a997f.